ci/jobs-dev/run_post_fv3r_ifi_missing_HERA.sh

```diff
@@ -0,0 +1,141 @@
+#!/bin/sh
+
+#SBATCH -o out.post.fv3r_ifi_missing
+#SBATCH -e out.post.fv3r_ifi_missing
+#SBATCH -J fv3r_ifi_missing
+#SBATCH -t 00:30:00
+#SBATCH -N 8 --ntasks-per-node=12
+##SBATCH -q debug
+#SBATCH -q batch
+#SBATCH -A ovp
+
+# specify computation resource
+export threads=1
+export MP_LABELIO=yes
+export OMP_NUM_THREADS=$threads
+export APRUN="srun"
+
+############################################
+# Loading module
+############################################
+module purge
+module use /contrib/spack-stack/spack-stack-1.8.0/envs/ue-intel-2021.5.0/install/modulefiles/Core
+module load stack-intel/2021.5.0
+module load stack-intel-oneapi-mpi/2021.5.1
+module load libpng/1.6.37
+module load jasper/2.0.32
+module load prod_util/2.1.1
+module load crtm/2.4.0.1
+module list
+
+msg="Starting fv3r_ifi_missing test"
+postmsg "$logfile" "$msg"
+
+export cmp_grib2_grib2=/home/Wen.Meng/bin/cmp_grib2_grib2_new
+export POSTGPEXEC=${svndir}/exec/upp_no_ifi.x
+
+# specify forecast start time and hour for running your post job
+export startdate=2023062800
+export fhr=010
+
+# specify your running and output directory
+export DATA=$rundir/fv3r_ifi_missing_${startdate}
+export tmmark=tm00
+rm -rf $DATA; mkdir -p $DATA
+cd $DATA
+
+export NEWDATE=`${NDATE} +${fhr} $startdate`
+                                                                                       
+export YY=`echo $NEWDATE | cut -c1-4`
+export MM=`echo $NEWDATE | cut -c5-6`
+export DD=`echo $NEWDATE | cut -c7-8`
+export HH=`echo $NEWDATE | cut -c9-10`
+
+
+cat > itag <<EOF
+&model_inputs
+fileName='$homedir/data_in/fv3r/dynf${fhr}.nc'
+IOFORM='netcdf'
+grib='grib2'
+DateStr='${YY}-${MM}-${DD}_${HH}:00:00'
+MODELNAME='FV3R'
+fileNameFlux='$homedir/data_in/fv3r/phyf${fhr}.nc'
+/
+&NAMPGB
+KPO=47,PO=1000.,975.,950.,925.,900.,875.,850.,825.,800.,775.,750.,725.,700.,675.,650.,625.,600.,575.,550.,525.,500.,475.,450.,425.,400.,375.,350.,325.,300.,275.,250.,225.,200.,175.,150.,125.,100.,70.,50.,30.,20.,10.,7.,5.,3.,2.,1.,
+write_ifi_debug_files=.true.
+/
+EOF
+
+
+rm -f fort.*
+
+#cp /nwprod/nam.v3.1.16/fix/nam_micro_lookup.dat ./eta_micro_lookup.dat
+cp ${svndir}/fix/nam_micro_lookup.dat ./eta_micro_lookup.dat
+
+# copy flat files instead
+cp ${svndir}/parm/postxconfig-NT-ifi.txt ./postxconfig-NT.txt
+
+cp ${svndir}/parm/params_grib2_tbl_new ./params_grib2_tbl_new
+
+#get crtm fix file
+for what in "amsre_aqua" "imgr_g11" "imgr_g12" "imgr_g13" \
+    "imgr_g15" "imgr_mt1r" "imgr_mt2" "seviri_m10" \
+    "ssmi_f13" "ssmi_f14" "ssmi_f15" "ssmis_f16" \
+    "ssmis_f17" "ssmis_f18" "ssmis_f19" "ssmis_f20" \
+    "tmi_trmm" "v.seviri_m10" "imgr_insat3d" "abi_gr" \
+    "ahi_himawari8" ; do
+    ln -s "${CRTM_FIX}/${what}.TauCoeff.bin" .
+    ln -s "${CRTM_FIX}/${what}.SpcCoeff.bin" .
+done
+
+for what in 'Aerosol' 'Cloud' ; do
```

Comment:
@SamuelTrahanNOAA Linking crtm fix files is not necessary since no simulated satellite products are generated in this test.

Reply:
I removed these links from the ifi scripts on all three platforms.